libs/libarchfpga/src/physical_types_util.cpp

```diff
@@ -0,0 +1,426 @@
+#include "vtr_assert.h"
+#include "vtr_memory.h"
+#include "vtr_util.h"
+
+#include "arch_types.h"
+#include "arch_util.h"
+#include "arch_error.h"
+
```

Comment:
Good to comment what the purpose of this file is / what routines should go in it.

Reply:
This description was added in the header file, I think that is a better place for this kind of comment as it has all the utils prototype.